storage/quota/quota.go

```diff
@@ -122,12 +121,21 @@ func StartQuotaDriver(dir string) (string, error) {
 
 // SetSubtree is used to set quota id for directory.
 func SetSubtree(dir string, qid uint32) (uint32, error) {
+	if isRegular, err := CheckRegularFile(dir); err != nil || !isRegular {
+		logrus.Debugf("set quota skip not regular file: %s", dir)
```

Comment:
I think this line of log is a little bit incorrect. 
When code  runs to this line, it happens to be CheckRegularFile's failure, right?
Then, if it is better to use `failed to set blablabla .....`.

Reply:
it is not failed to set set quota, just to skip the unregular file.